### PR TITLE
Attendance file for florie

### PR DESCRIPTION
**Pull request by ghost**
Finally, my attendance file !

### CODE REVIEW

**Comment by JacintheR** [comment]
LGTM

**Comment by bluciam** [comment]
Yay!

**Comment by JacintheR** [comment]
:ship:

File: FlorieAttendance [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Hi
+
+I am Florie Lambrey
```
Comment (JacintheR) [inline FlorieAttendance:3]: Hi Florie!

File: FlorieAttendance [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
```
Comment (bluciam) [inline FlorieAttendance:4]: Hey! I<m Francoise :+1:

File: FlorieAttendance [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+Here is my attendance file
+
+You can find me at :
+@FlorieLambrey
+
+Bye :-)
```
Comment (CleoQc) [inline FlorieAttendance:10]: À la prochaine!

**Comment by NadiaTahiri** [comment]
@bluciam super cool ☺

**Comment by nastaranj** [comment]
:100:

**Comment by bluciam** [comment]
Thanks! @NadiaUQAM

**Pull request merged**